conn.go

```diff
@@ -955,14 +955,39 @@ func (c *Conn) WriteResponse(code int, enhCode EnhancedCode, text ...string) {
 		}
 	}
 
+	// Multiple tcp packets cause error
+	// in some controllers like sim800. reply must be a single tcp packet. 14.09.2020 ignatev vadim.
+	// example:
+	// s: 220 greets
+	// c: EHLO me
+	// s: 250-hello-me
+	// s: 250-SIZE 1111
+	// s: 250-AUTH LOGIN
+	// c: AUTH LOGIN      <- client got previous packet and began authorization
+	// s: 250-STARTTLS    <- server keeps sending reply to EHLO
```

Comment:
Why client ends up assuming the list of capabilities is completed? It should wait for `250 ` (without dash). This is clearly a bug in client code and I believe go-smtp should not be monkey-patched for every single broken client implementation like that.

Though I suppose sending a capability list as a single segment might be considered a general optimization (likely an unnecessary one), in this case...